packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -24,6 +24,9 @@
     "Debuggers"
   ],
   "dependencies": {
+    "@salesforce/apex-node": "0.1.4",
+    "@salesforce/core": "2.11.0",
```

Comment:
We'll need to update this extension's `.vscodeignore` to add all the jsforce, sfdx-faye & dayjs paths to ignore https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/.vscodeignore